test/runtimes/runner/lib/lib.go

```diff
@@ -103,10 +111,14 @@ func getTests(ctx context.Context, d *dockerutil.Container, lang, image string,
 		for _, tc := range indices[i:end] {
 			// Add test if not excluded.
 			if _, ok := excludes[tests[tc]]; ok {
-				log.Infof("Skipping test case %s\n", tests[tc])
+				log.Infof("Skipping test case %s", tests[tc])
 				continue
 			}
-			tcs = append(tcs, tests[tc])
+			// Check that it is allowed.
+			if filterRe.MatchString(tests[tc]) {
+				log.Infof("Including test case %s", tests[tc])
```

Comment:
I think we should either remove this log or make it a debug log, otherwise this will likely create a lot of noise. Java for instance has 4K tests IIRC.